src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/TplEventSource.cs

```diff
@@ -373,7 +373,7 @@ public void TaskWaitContinuationComplete(int TaskID)
         /// </summary>
         /// <param name="TaskID">The task ID.</param>
         [Event(TASKWAITCONTINUATIONSTARTED_ID,
-         Level = EventLevel.Verbose, Keywords = Keywords.TaskStops)]
+         Level = EventLevel.Verbose, Keywords = Keywords.Tasks)]
```

Comment:
It might be worth double-checking whether this is going to break any tooling in Visual Studio, i.e. that Tasks is being specified in addition to TaskStops.

Reply:
Will do. Thanks!